Package.swift

```diff
@@ -14,6 +14,24 @@
 import Foundation
 import PackageDescription
 
+// MARK: - Parse build arguments
+
+func hasEnvironmentVariable(_ name: String) -> Bool {
```

Comment:
You can also update the `SWIFTCI_USE_LOCAL_DEPS` env var lookup at the bottom of the file to use this.